doc/nrf/test_and_optimize/test_framework/running_unit_tests.rst

```diff
@@ -51,9 +51,10 @@ To generate the test project and run the unit tests, locate the directory with t
 
    .. group-tab:: west
 
-      .. code-block:: console
+      .. parsed-literal::
```

Comment:
Shouldn't the line <board_target> on no:31 and 45 also change to *board_target*?